cosmic-panel-bin/src/minimize.rs

```diff
@@ -9,7 +9,8 @@ use crate::space_container::SpaceContainer;
 
 #[derive(Debug, Clone)]
 pub struct MinimizeApplet {
-    pub priority: i32,
+    pub priority: u32,
```

Comment:
While I think this *should* work, I don't think the panel should send this `MinimizeApplet` data for applets that don't have the highest priority in a panel anyway, so it doesn't need to be included here imo.

Reply:
Fixed